app/services/workflow_service/pensioner_capital_disregard.rb

```diff
@@ -0,0 +1,56 @@
+module WorkflowService
+  class PensionerCapitalDisregard
+    def initialize(particulars)
+      @particulars = particulars
+      @submission_date = @particulars.request.meta_data.submission_date
+      @thresholds = Threshold.value_for(:pensioner_capital_disregard, at: @submission_date)
+    end
+
+    def value
+      return 0 unless pensioner?
+
+      return passported_value if passported?
+
+      raise 'No disposable income specified for non-passported applicant' if disposable_income.nil?
+
+      disregard_value(disposable_income)
+    end
+
+    private
+
+    def pensioner?
+      earliest_dob_for_pensioner
+      @submission_date - minimum_pensioner_age.years >= applicant_dob
```

Comment:
Should these two lines just be `earliest_dob_for_pensioner >= applicant_dob`?

Reply:
👍